### DISCUSSION TITLE
Kiali-Operator Helm Chart - Values File - Repo and Tag not working with Argo-CD

### DISCUSSION THREAD

**Discussion by L-U-C-K-Y** [kiali/kiali#4337] [Q&A] [answered]
Hi all 👋🏼

I am trying to deploy the kiali-operator only and have performed the following steps:

1. Cloned the `kiali/helm-charts` repo from here https://github.com/kiali/helm-charts with the tag v1.37.0 (compatible with istio 1.10)
2. Copied the `kiali-operator` folder into my helm-charts repo that is connected to `argo-cd`
3. Defined the below config in the `values.yaml`

```bash
image:
  repo: kiali-operator/kiali-operator
  tag: 1.37.0

# ERROR: Back-off pulling image "kiali-operator/kiali-operator:1.37.0"

image:
  repo: kiali.org/helm-charts
  tag: 1.37.0

# ERROR: Back-off pulling image "kiali.org/helm-charts:1.37
```

Appreciate any help!

**Comment by jmazzitelli** [comment, marked as answer]
The tags need to be one of the tags in quay.io - here: https://quay.io/repository/kiali/kiali-operator?tab=tags
You are missing the "v" prefix.

**Reply by jmazzitelli** [reply]
Note also you did not put a repo in your image name so it will default to docker.io - we do not publish on docker.io - we publish on quay.io. Also, the organization name is wrong too (its `kiali` not `kiali-operator` ... so its `kiali/kiali-operator` not `kiali-operator/kiali-operator`). So, basically, you have everything wrong :-D

The operator image we publish is `quay.io/kiali/kiali-operator:v1.37` and the server image is `quay.io/kiali/kiali:v1.37`.

See: https://quay.io/repository/kiali/kiali?tab=tags and https://quay.io/repository/kiali/kiali-operator?tab=tags

**Reply by L-U-C-K-Y** [reply]
Many thanks for your detailed answer, was able to deploy the operator!

I created a pull request that would have helped me. If it's helpful, feel free to merge.

**Reply by israel-hdez** [reply]
@L-U-C-K-Y can you post the link to you pull request?
I'm asking, because I am unable to find it.

**Reply by L-U-C-K-Y** [reply]
@israel-hdez sure: https://github.com/kiali/helm-charts/pull/103
